src/algebra/ordered_group.lean

```diff
@@ -763,23 +957,15 @@ lemma abs_pos_of_pos (h : 0 < a) : 0 < abs a := abs_pos.2 h.ne.symm
 
 lemma abs_pos_of_neg (h : a < 0) : 0 < abs a := abs_pos.2 h.ne
 
-lemma abs_sub (a b : α) : abs (a - b) = abs (b - a) :=
-by rw [← neg_sub, abs_neg]
-
-lemma abs_le' : abs a ≤ b ↔ a ≤ b ∧ -a ≤ b := max_le_iff
-
-lemma abs_le : abs a ≤ b ↔ - b ≤ a ∧ a ≤ b :=
-by rw [abs_le', and.comm, neg_le]
-
-lemma neg_le_of_abs_le (h : abs a ≤ b) : -b ≤ a := (abs_le.mp h).1
-
-lemma le_of_abs_le (h : abs a ≤ b) : a ≤ b := (abs_le.mp h).2
-
-lemma le_abs : a ≤ abs b ↔ a ≤ b ∨ a ≤ -b := le_max_iff
-
-lemma le_abs_self (a : α) : a ≤ abs a := le_max_left _ _
-
-lemma neg_le_abs_self (a : α) : -a ≤ abs a := le_max_right _ _
+/-  I (DT) introduced this lemma to prove `sub_le_sub_flip` below.  Now I wonder what is the
+point of either of these lemmas... -/
```

Comment:
The lemma could have its uses. It was introduced in #6720, but I'm not sure if it is used there. I think it can be removed if it is unused in the library (@semorrison ?)

Reply:
These lemmas moved to lines 747-765 of the same file `algebra/ordered_group.lean`.

Reply:
Yeah, I'm fine leaving it for now